pkg/network/setup/netstat.go

```diff
@@ -375,3 +392,21 @@ func filterHostDevicesByAlias(hostDevices []api.HostDevice, prefix string) []api
 	}
 	return filteredHostDevices
 }
+
+func splitIPByFamiliy(ips []string) ([]string, []string) {
+	var IPv4Addresses []string
+	var IPv6Addresses []string
+
+	for _, ipRaw := range ips {
+		ip := net.ParseIP(ipRaw)
+		switch {
+		case ip == nil:
+			continue
```

Comment:
do we want log in this case ?
(lower verbosity)
or we don't care for such cases

Reply:
I am not in favor of logging so deep in the stack, it is IMO a bad practice that should be avoided.
But you are encouraged to open an offline discussion about logging if you want.

If you want some background on my reasoning, visit the blog post [1].

[1] https://ehaas.net/blog/logs-and-errors/

Reply:
Thanks, right, it poulates the log file, and make debugging harder - this can be a nice effort to go over sig-network code and makes the logs friendly to the personal that need debugging if desired